x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ingest_pipelines/ingest_pipeline_flyout.tsx

```diff
@@ -82,6 +85,44 @@ export const IngestPipelineFlyout: React.FC<IngestPipelineFlyoutProps> = ({
       </EuiFlyoutHeader>
       <EuiFlyoutBody>
         <EuiFlexGroup direction="column" gutterSize="none">
+          <EuiFlexItem>
+            {extractionDisabled ? (
+              <EuiCallOut
+                title={i18n.translate(
+                  'xpack.enterpriseSearch.content.index.pipelines.settings.extractBinaryDisabledWarningTitle',
+                  {
+                    defaultMessage: 'Content extraction cannot be configured',
+                  }
+                )}
+                color="warning"
+                iconType="warning"
+              >
+                <p>
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.content.index.pipelines.settings.extractBinaryDisabledWarningContent',
+                    {
+                      defaultMessage:
+                        "Because local content extraction has been enabled in this connector's configuration, pipeline content extraction settings cannot be used.",
```

Comment:
```suggestion
                        "Because local content extraction is enabled for this connector, pipeline content extraction settings cannot be used.",
```

tiny copy tweak

Reply:
Cheers!